src/lua/LuaBodyComponent.h

```diff
@@ -0,0 +1,35 @@
+// Copyright © 2008-2022 Pioneer Developers. See AUTHORS.txt for details
```

Comment:
It seems this file is not included anywhere.

Reply:
Correct - none of the existing components have a Lua binding so this is currently unused. It should be included by whatever file calls `BodyComponentDB::RegisterLuaInterface<T>(typeName)`. This file is separate to avoid dragging in the LuaObject dependency into the BodyComponent header (and thus into most of our C++ files).